pkg/cmdimpl/add.go

```diff
@@ -80,14 +87,20 @@ func updateParametersIfNecessary() error {
 	}
 
 	if params.Url == "" {
-		urlout, err := exec.Command("git", "remote", "get-url", "origin").CombinedOutput()
+		gitClient := git.New(nil)
+		repo, err := gitClient.Open(params.Dir)
+		if err != nil {
+			return err
```

Comment:
it would be helpful for debbuging if we have e more detailed error description. Maybe something like fmt.Errof("error accessing %s directory %s`, params.Dir,err) or similar.